animations.md

```diff
@@ -1,311 +1,377 @@
+  * <code>[flutter\_sprites/flutter\_sprites](http://docs.flutter.io/flutter_sprites/flutter_sprites/flutter_sprites-library.html)</code>:
+  Provides APIs for game development.
+
+The rest of this documentation covers how to use the APIs from the
+<code>[flutter/animation](http://docs.flutter.io/flutter/animation/animation-library.html)</code>
+library.
+
+Animations in Flutter are encapsulated as
+<code>[Animation](http://docs.flutter.io/flutter/animation/Animation-class.html)</code>
+objects that contain these key properties:
+
+  * <strong>Value</strong>: The animation object’s typed value represents a data point within a
+  range (also called an <em>interpolation</em>). You can define an animation class to generate
+  values based on a given functional mapping, such as a linear, curve, or step-wise function.
+  * <strong>Status</strong>: The animation object’s current status (such as <em>forward</em>,
+  <em>backward</em>, <em>completed</em>, and <em>dismissed</em>), as listed by the constants of
+  the
+  <code>[AnimationStatus](http://docs.flutter.io/flutter/material/AnimationStatus-class.html)</code>
+  class.
+
+Typically, the animation’s value and status changes over time as it runs. You
+can attach an animation object to a widget or listen for changes to the
+animation object. Based on changes to the animation object’s properties, the
+framework can modify the way your widget appears and rebuild the widget tree to
+produce visual transitions such as sliding, scaling, rotation, height or width
+changes, and fading.
+
+This flexible model allows you to define interpolations for spatial
+positioning, color gradients, image scaling, and others. You can define your
+own subclass of
+<code>[Animation](http://docs.flutter.io/flutter/animation/Animation-class.html)</code>
+or use an animation subclass that the framework provides.
+
+### Controlling how animations run
+
+This section briefly describes some of the common APIs you will use for
+creating animation effects and controlling how they run.
+
+#### [AnimationController](http://docs.flutter.io/flutter/material/AnimationController-class.html)
+
+Use this class to specify how the animation should run. The
+<code>[AnimationController](http://docs.flutter.io/flutter/material/AnimationController-class.html)</code>
+class lets you define important characteristics of the animation, such as its
+duration and playback direction (forward or reverse).
+
+The following snippet shows how you might create an animation that runs for 500 milliseconds:
+
+
+<pre class=prettyprint>
+final AnimationController controller =
+    new AnimationController(duration: const Duration(milliseconds: 500));
+</pre>
+
+#### [CurvedAnimation](http://docs.flutter.io/flutter/material/CurvedAnimation-class.html)
+
+Use this class to apply a non-linear curve to an animation object it wraps. By
+applying different curve types, you can visually modify the spatial movement
+pattern for the wrapped animation object. The
```

Comment:
A curve (usually) modifies the output of the AnimationController, adjusting the the graph. This may or may not have anything at all to do with position.